target/linux/ramips/dts/mt7621_netgear_wax214v2.dts

```diff
@@ -0,0 +1,226 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "netgear,wax214v2", "mediatek,mt7621-soc";
+	model = "Netgear WAX214v2";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_amber;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_blue;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 10 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_green: power_green {
+			label = "green:power";
```

Comment:
Please use `function` && `color` properties instead of the `label`. examples:
https://git.openwrt.org/?p=openwrt%2Fopenwrt.git&a=search&h=HEAD&st=grep&s=function+%3D+LED

Reply:
Done, except for LEDs green:wifia and green:wifin for which I could not find suitable LED function macros.

Reply:
`function = LED_FUNCTION_WLAN` and use `function-enumerator` to differentiate.

Reply:
Done - it appears that functions LED_FUNCTION_WLAN_2GHZ and LED_FUNCTION_WLAN_5GHZ have since been added, so I used those. LED entries appear like this now:

```
root@netgear-wax214v2:~# ls -1 /sys/class/leds/
amber:power
blue:power
green:power
green:wlan-2ghz
green:wlan-5ghz
mt76-phy0
mt76-phy1
```